src/HotChocolate/Core/src/Execution/Options/IFusionOptionsAccessor.cs

```diff
@@ -0,0 +1,22 @@
+using System.Diagnostics;
+
+namespace HotChocolate.Execution.Options;
+
+public interface IFusionOptionsAccessor
```

Comment:
Lets not put it into the core and move it into fusion... Lets create a separate options object.

Reply:
I think we should just add an object called `FusionOptions` into the fusion project. We can add a separate ModifyOptions extension method on the fusion builder.

Reply:
Yes, I moved it here since the options seemed to be tied to the executor and I just wanted to reuse existing functionality.